site/src/pages/TemplatesPage/MultiOrgTemplatePage/TemplatesPageView.tsx

```diff
@@ -63,89 +63,168 @@ export const TemplatesPageView: FC<TemplatesPageViewProps> = ({
   canCreateTemplates,
   error,
 }) => {
+  const sidebarWidth = 208;
   const navigate = useNavigate();
   const [urlParams] = useSearchParams();
   const isEmpty = templatesByOrg && templatesByOrg["all"].length === 0;
   const activeOrg = urlParams.get("org") ?? "all";
   const visibleTemplates = templatesByOrg
     ? templatesByOrg[activeOrg]
     : undefined;
+  const hasMultipleOrgs = Boolean(
```

Comment:
@Emyrk can you squint at this?

Reply:
This assumes there is a template in each org. If an org has 0 templates, should we show that org with a `(0)`?